### PR TITLE
polish and fix contributing.md

### PR DESCRIPTION
**Pull request by MiroDudik**
Signed-off-by: Miro Dudik <mdudik@gmail.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,68 +2,68 @@
...
+Reduction constructors require a parameter corresponding to an estimator that implements the `fit` method with the `sample_weight` argument. Parity constraints for reductions are expressed via instances of various subclasses of the class `fairlearn.reductions.Moment`. Formally, instances of the class `Moment` implement vector-valued random variables whose sample averages over the data are required to be bounded (above and/or below).
 
 ```python
 constraints = Moment()
-reduction = Reduction(estimator, objective=objective, constraints=constraints, **kwargs)
+reduction = Reduction(estimator, constraints)
 ```
 
-Reduction-based disparity mitigation algorithms (such as the ones under `fairlearn.reductions`) provide `fit`, `predict`, and `_pmf_predict` methods with the following signatures:
+Reductions  provide `fit`, `predict`, and `_pmf_predict` methods with the following signatures:
```
Comment (riedgar-ms) [inline CONTRIBUTING.md]: Do we want to publicise `_pmf_predict` or should we keep that private for now?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,68 +2,68 @@
...
+All of the currently supported parity constraints (subclasses of `Moment`) are based on sensitive features that need to be provided to `fit` as a keyword argument `sensitive_features`. In the future, we will also allow providing sensitive features as columns of `X`.
 
-### Post-processing methods
+#### Post-processing algorithms
 
-Post-processing methods require an already trained predictor. For consistency we also provide the option to pass an estimator instead, and will call `fit` internally. For post-processing methods we provide the `constraints` argument in the form of a string.
+The constructors of post-processing algorithms require an already trained predictor as an argument. For consistency, we also allow providing an estimator instead, and the estimator is then fitted on the data at the beginning of the execution of `fit`. For post-processing algorithms, the `constraints` argument is provided as a string.
```
Comment (riedgar-ms) [inline CONTRIBUTING.md]: I'm not fond of "allow providing" as a phrase.

**Comment by riedgar-ms** [comment]
Will this also go into v0.3.0?

File: CONTRIBUTING.md (hunk 2 above)
Comment (MiroDudik) [reply CONTRIBUTING.md]: rephrased

File: CONTRIBUTING.md (hunk 1 above)
Comment (MiroDudik) [reply CONTRIBUTING.md]: good point. removing until we standardize.

**Pull request merged**